### DISCUSSION TITLE
use the contents area

### DISCUSSION THREAD

**Discussion by miso1459** [oqtane/oqtane.framework#1287] [Q&A]
Dear everyone.

I want to use the full area. not scroll. resizable

Help me.

Thank you.

![image](https://user-images.githubusercontent.com/37363887/116176840-c424d280-a74d-11eb-9829-ab70c529772e.png)

**Comment by leigh-pointer** [comment]
Hello @miso1459 You will need to create a Theme.  you can find the default themes for Oqtane in the client source code project.  The framework will also create you the scaffolding for creating a new theme in the Theme managment page.

**Reply by miso1459** [reply]
Thank you.

**Comment by sbwalker** [comment]
@leigh-pointer I am not so sure that this is not a bug. In the OqtaneTheme, the Container.razor component includes Bootstrap "container" classes which result in left and right margins around the content. If you remove these "container" classes and you place the module in the Top Full Width Pane - it will expand to fill the entire area ( and result in a horizontal scroll bar - which is caused by the Bootstrap "row" class - which would also need to be fixed ). I am thinking that these "row" and "container" classes should not be embedded in Container.razor?

**Reply by leigh-pointer** [reply]
@sbwalker I have looked at this and found the issue. few more tests.

**Reply by miso1459** [reply]
Thank you.
